app/bundles/UserBundle/Controller/PublicController.php

```diff
@@ -37,18 +37,16 @@ public function passwordResetAction()
                 $data = $form->getData();
                 $user = $model->getRepository()->findByIdentifier($data['identifier']);
 
-                if (null == $user) {
-                    $form['identifier']->addError(new FormError($this->translator->trans('mautic.user.user.passwordreset.nouserfound', [], 'validators')));
```

Comment:
Please remove the `mautic.user.user.passwordreset.nouserfound` message from the ini file too as you removed all its usages.